MdeModulePkg/Library/UefiBootManagerLib/BmBoot.c

```diff
@@ -1505,7 +1509,7 @@ BmExpandLoadFile (
 
   Status = LoadFile->LoadFile (LoadFile, FilePath, TRUE, &BufferSize, FileBuffer);
   if (EFI_ERROR (Status)) {
-    FreePages (FileBuffer, EFI_SIZE_TO_PAGES (BufferSize));
+    FreeAlignedPages (FileBuffer, EFI_SIZE_TO_PAGES (BufferSize));

```

Comment:
I think we also need to change this to FreeAlignedPages, also? https://github.com/microsoft/mu_basecore/blob/release/202302/MdeModulePkg/Library/UefiBootManagerLib/BmBoot.c#L1416C12-L1416C12. Not sure if there is another destruction path that needs to be updated.

Reply:
Done, I think that's it.  Note that the body for FreeAlignedPages is a carbon copy of FreePages, so I'm not sure we'd get any kind of runtime error in case of mismatch, which is a bit unfortunate.

Reply:
I had a suspicion that was the case, but didn't check it :).